services/src/main/java/org/apache/druid/cli/CliCoordinator.java

```diff
@@ -233,10 +233,14 @@ public LoadQueueTaskMaster getLoadQueueTaskMaster(
               ZkPathsConfig zkPaths
           )
           {
+            int poolSize = Integer.parseInt(properties.getProperty(
```

Comment:
Could we place that parameter into `DruidCoordinatorConfig`?

Reply:
This parameter would be just used here and wouldn't be called in other classes, so there is no need to hold it in `DruidCoordinatorConfig`.